components/icon-button/index.js

```diff
@@ -23,6 +23,8 @@ class IconButton extends Component {
 	render() {
 		const { icon, children, label, className, tooltip, focus, ...additionalProps } = this.props;
 		const classes = classnames( 'components-icon-button', className );
+		const tooltipText = tooltip || label;
+		const showTooltip = !! tooltip || ( label && ! children && false !== tooltip );
```

Comment:
Edge case: Should `showTooltip` be `true` or `false` if `children` is `[]` ?

(Related: https://reactjs.org/docs/react-api.html#reactchildrencount )

Reply:
mmm, I guess it's better if it's true.